autonomic-administration-plugin/src/main/java/br/com/autonomiccs/autonomic/administration/plugin/AdministrationAgent.java

```diff
@@ -202,16 +211,16 @@ private void workOnCluster(ClusterVO cluster, ClusterAdministrationHeuristicAlgo
      * method; then it shutdown idle hosts with (if possible)
      * {@link #shutdownIdleHosts(ClusterVO, ClusterAdministrationHeuristicAlgorithm)} method.
      */
-    private void processCluster(ClusterVO cluster, ClusterAdministrationHeuristicAlgorithm administrationAlgorithm) {
+    protected void processCluster(ClusterVO cluster, ClusterAdministrationHeuristicAlgorithm administrationAlgorithm) {
         long clusterId = cluster.getId();
         if (!autonomicClusterManagementService.canProcessCluster(clusterId, administrationAlgorithm)) {
```

Comment:
@alexandrelimassantana I think that we can remove this check (line 216) and leave just the one at line 177. I am concerned that it might be some case that we needed to cover, but now I can't see any reason to keep that check. What do you think @rafaelweingartner?